shell/platform/fuchsia/runtime/dart/profiler_symbols/dart_profiler_symbols.dart

```diff
@@ -79,12 +77,12 @@ ${parser.usage};
 }
 
 class Symbol {
```

Comment:
It looks like we have all the info we need to set all these fields at construction-time, so should we add a constructor that takes offset, size and name and remove the `late`s?

Reply:
agreed. fixed.